### DISCUSSION TITLE
Flowise Error

### DISCUSSION THREAD

**Discussion by AidanGilchrist** [milvus-io/milvus#32622] [Q&A and General discussion]
I'm trying to switch a flowise chat from pinecone to milvus to allow for upserting. I've set milvus up in an aws server and am able to call it correctly, I keep getting the error in the attached image. Has anyone experienced this or have an idea how to fix it.
![Screenshot 2024-04-25 113324](https://github.com/milvus-io/milvus/assets/122918762/41101fa9-f82b-4c58-9024-075afe00be53)
Edit: I am able to call the server within python and the command line correctly.

**Comment by xiaofan-luan** [comment]
seems that there is a mismatch.
when doing upsert, you have to give primarykey.
On the other side, if you don't care about primarykey, use insert with auto key generation should work.

@AidanGilchrist Could you share us which document you are following so we can help?

**Reply by robert-sarosi** [reply]
I am experiencing the same issue with this minimal setup. Latest milvus is running in docker. Could someone give any guidance on this? I am new to milvus.


<img width="1769" alt="Screenshot 2024-05-07 at 09 26 21" src="https://github.com/milvus-io/milvus/assets/162968372/2fac48f6-4347-44c7-9d1b-31181a3805eb">
<img width="666" alt="Screenshot 2024-05-07 at 09 26 28" src="https://github.com/milvus-io/milvus/assets/162968372/618d3a0c-922c-4530-819b-8f8981a1d220">
<img width="1823" alt="Screenshot 2024-05-07 at 09 28 10" src="https://github.com/milvus-io/milvus/assets/162968372/e2831e33-260c-4838-9cf6-6258ccb80e69">

**Comment by xiaofan-luan** [comment]
@shanghaikid will help on fix this issue on flowise side

**Comment by zhanshuyou** [comment]
@AidanGilchrist 
It looks like the error is caused by the id field in the inserted data. Milvus probably has autoId enabled, so you can try removing the id field from the inserted data.
